packages/devtools_app/lib/src/profiler/cpu_profile_controller.dart

```diff
@@ -87,6 +89,20 @@ class CpuProfilerController {
     }
   }
 
+  @override
+  List<CpuStackFrame> matchesForSearch(String search) {
+    if (search == null || search.isEmpty) return [];
+    final matches = <CpuStackFrame>[];
+    final caseInsensitiveSearch = search.toLowerCase();
+    final currentStackFrames = _dataNotifier.value.stackFrames.values;
+    for (final frame in currentStackFrames) {
+      if (frame.name.toLowerCase().contains(caseInsensitiveSearch)) {
```

Comment:
we're adding a lot of toLowerCase calls here and elsewhere. Maybe add caseInsentiveMatch helper to the utils library. At some point it could make sense to do the match a character at a time to avoid the toLowerCase() and that would make it easy to optimize in the future.